site/docs/v1/tech/samlv2/_saml_limitations.adoc

```diff
@@ -1,15 +1,22 @@
-FusionAuth supports SAML both as an Identity Provider (IdP) and as a Service Provider (SP). 
+FusionAuth supports SAML both as an Identity Provider (IdP) and as a Service Provider (SP). The IdP implementation has certain limitations. 
 
-The IdP implementation has certain limitations. Version `2.0` is supported; other SAML versions are unsupported.
+Version `2.0` is supported; other SAML versions are unsupported.
 
-FusionAuth supports two `NameIDPolicy` values:
+Please examine the documented `NameIDPolicy` of the software package for which you are configuring FusionAuth to act as the IdP to determine the correct value. FusionAuth won't fail validation on any `NameIDPolicy` value, however if the requested `NameIDPolicy` is `persistent`, FusionAuth will use that, otherwise it will use `email` for the name Id.
```

Comment:
Should we spell out which property is used for these different name Id formats? 

> ... will use that, and return the User's unique Id (`user.id`), otherwise the user's email address (`user.email`) will be used for the name Id.